packages/bbui/src/Form/Core/DatePicker.svelte

```diff
@@ -23,6 +23,15 @@
   let open = false
   let flatpickr, flatpickrOptions
 
+  // Another classic flatpickr issue. Errors were randomly being thrown due to
+  // flatpickr internal code. Making sure that "destroy" is a valid function
+  // fixes it. The sooner we remove flatpickr the better.
```

Comment:
😅

Reply:
Maybe next lab day I'll do something about this :joy: